### ISSUE TITLE
[Feature request] Analyse time of likes

### ISSUE THREAD

**Issue by lukapusic** [x0rz/tweets_analyzer#22]
Some people tweet less, but like lots of stuff. Likes can be used to determine at what hours of day accounts are active.

**Comment by x0rz** [comment]
Yep, that's something I though too. One issue might be the limitation (only 75 requests = 1500 last favorites https://dev.twitter.com/rest/reference/get/favorites/list)
I will look into this 👍
